contracts/bonding/libraries/TranscoderPool.sol

```diff
@@ -0,0 +1,372 @@
+pragma solidity ^0.4.17;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/*
+ * @title A pool of registered transcoders and their stakes implemented as a sorted doubly linked list
+ */
+library TranscoderPool {
```

Comment:
General comment about the transcoder pool is that I think that the implementation is a little bit verbose and potentially confusing because we introduce transcoder specific terminology. In reality however, are we doing anything at all that is unique to transcoders...or is it just a sorted doubly linked list data structure, using a node's value as the sort key? With the ability for the node's value to be updated triggering a resort?

If this is actually the case, the implementation can potentially avoid all the `better`, `worse`, `transcoder` sepcific terminology, and just use `next`, `prev`, and update `value` functions?

Open to discussion on this. It may be more auditable, reusable, and convenient.